InfluxData.Net.InfluxDb/Helpers/SerieExtensions.cs

```diff
@@ -26,6 +27,52 @@ public static T FirstRecordValueAs<T>(this Serie serie, string columnKey)
         }
 
         /// <summary>
+        /// Converts an ubiquitous enumeration of series to a strongly typed enumeration by matching property names.
+        /// </summary>
+        /// <typeparam name="T">Type to convert the enumeration of series to</typeparam>
+        /// <param name="series">Series to convert</param>
+        /// <returns>Strongly typed enumeration representing the series</returns>
+        public static IEnumerable<T> RecordsAs<T>(this IEnumerable<Serie> series)
+         where T : new()
+        {
+            if (series == null)
+                yield return default(T);
+
+            Type t = typeof(T);
+
+            foreach (var serie in series)
+            {
```

Comment:
@pootzko One thing worth noting is that even though it iterates through all series the extensions expects each serie to match the structure of the type provided in 'T'. I'll see if I can come up with a solution to support different structured series. Cheers

Reply:
I think that's fine. The end-user should take care of this in my opinion. The only situation where it could get a bit mess is if someone is using the MultiQuery to fetch different series.